playbooks/tasks/portal-role-task-mongo-replicaset-get-primary-sub-task2.yml

```diff
@@ -0,0 +1,75 @@
+---
+
+# Find primary node in the MongoDB replicaset (subtask to loop over)
+# TODO: This ansible playbook task, should be moved to portal role
+
+# Notes:
+# - This task file is expected to be called in a loop over MongoDB replicaset
+#   member hosts with loop item variable {{ host_item }}.
+# - These tasks should be generally delegated to {{ host_item }} host, not run
+#   on the current Ansible host that we are handling.
+# - For subcrap testing we must set ansible_port according to the subcrap host.
+
+# Search for MongoDB primary host until you find it
+
+- name: Check that mongo container is running
+  community.docker.docker_container_info:
+    name: mongo
+  delegate_to: "{{ host_item }}"
+  vars:
+    ansible_port: "{{ hostvars[host_item].ansible_port | default(22) }}"
```

Comment:
just going to put the one comment here, but 22 could be a variable right?

Reply:
Extracted to `default_ssh_port`.